vendor/assets/javascripts/syntaxhighlighter_lib/shCore.js

```diff
@@ -385,7 +385,8 @@ dp.sh.Highlighter.prototype.SwitchToList = function() {
     var li = this.CreateElement('LI');
     var span = this.CreateElement('SPAN');
 
-    span.innerHTML = lines[i] + '&nbsp;';
+    if(lines[i] === "") { span.innerHTML = lines[i] + '&nbsp;'; }
```

Comment:
Put the body of if/else branches on separate lines after the if/else. Always use `{ }` to delimit bodies (i.e., don't do single-line branches).

Also, in the `if` case the `lines[i]` is empty, so doesn't need to be included when setting `span.innerHTML`.